checkout-ui-custom/src/controller/CollectController.js

```diff
@@ -339,8 +339,10 @@ const CollectController = (() => {
           const savingCollect = localStorage.getItem('saving-shipping-collect');
 
           if (!savingCollect) {
+            const { phone } = console.log(getOrderFormCustomData(PICKUP_APP))

```

Comment:
are you intentionally destructuring out of a console log?

Reply:
Nope, thank you